lib/sort_by_length.rb

```diff
@@ -1,7 +1,23 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2) n = array.length
+# Space complexity: O(1) only one array is being made
 def sort_by_length(my_sentence)
```

Comment:
Overall nice implementation of BubbleSort.  However it's O(n) space complexity since you do `my_sentence.split`.